src/App.js

```diff
@@ -1,11 +1,31 @@
-// import logo from './logo.svg';
+import React from 'react';
 import Calculator from './componets/calculator';
 import './App.css';
+import Quotes from './componets/qoutes';
 
 function App() {
+  const [quote, setQuote] = React.useState(['loading...']);
+  const [result, setResult] = React.useState({ total: null, next: null, operation: null });
+  React.useEffect(() => {
+    setQuote('loading...');
+    fetch('https://api.api-ninjas.com/v1/quotes?category=computers', {
+      method: 'GET',
+      headers: {
+        'X-API-Key': 'OAcvP3F39QWUffY523tlRg==3Y15nTus6njhSQ1w',
+      },
+    })
+      .then((response) => response.json())
+      .then((data) => {
+        const res = `${data[0].quote} - ${data[0].author}`;
+        setQuote(res);
+      })
+      .catch((error) => setQuote(['error ', error.message, ' quote']));
+  }, [result]);
```

Comment:
- [OPTIONAL] To avoid confusion for the user, it is advisable not to link the change to the quote with the result of the calculator. This is because the two elements have different functionalities, and linking them could make it difficult for users to understand the purpose of each feature. Therefore, it would be beneficial to keep them separate for a better user experience.

Reply:
Noted! thanks for the review

Reply:
fixed